handsontable/src/editors/autocompleteEditor/autocompleteEditor.js

```diff
@@ -127,11 +183,47 @@ export class AutocompleteEditor extends HandsontableEditor {
           }
         }
 
+        if (rootInstanceAriaTagsEnabled) {
+          setAttribute(TD, [
+            A11Y_OPTION(),
+            // Add `setsize` and `posinset` only if the source is an array.
+            ...(sourceArray ? [A11Y_SETSIZE(sourceSize)] : []),
+            ...(sourceArray ? [A11Y_POSINSET(sourceArray.indexOf(value) + 1)] : []),
+            ['id', `${this.htEditor.rootElement.id}_${value}`],
```

Comment:
I wonder if adding a cell value to the id attribute is necessary. The row and col would not be enough?

![image](https://github.com/handsontable/handsontable/assets/571316/7447962d-264f-46fe-a524-fb1698446f80)

Reply:
Row and col identify the editor in the main table, but I think I can change the value in the id to row and col in the dropdown HOT instance 👍